src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
```

Comment:
Consider whether `firstLeg` and `secondLeg` should be of type `int`. In real-world scenarios, the lengths of sides are not always whole numbers, so using `double` might be more appropriate.